gsy_framework/utils.py

```diff
@@ -306,51 +333,51 @@ def get_area_uuid_name_mapping(area_dict, results):
 
 
 def round_floats_for_ui(number):
+    """Round the given number using the scale required by the UI."""
     return round(number, 3)
 
 
 def round_prices_to_cents(number):
+    """Round the given number using 2-digits scale (to represent cents)."""
     return round(number, 2)
 
 
-def create_subdict_or_update(indict, key, subdict):
-    if key in indict:
-        indict[key].update(subdict)
+def add_or_create_key(dictionary, key, value):
+    """Increase the amount at the given key by the provided value."""
+    if key in dictionary:
+        dictionary[key] += value
     else:
-        indict[key] = subdict
-    return indict
+        dictionary[key] = value
+    return dictionary
 
 
-def add_or_create_key(dict, key, value):
-    if key in dict:
-        dict[key] += value
+def subtract_or_create_key(dictionary, key, value):
+    """Decrease the amount at the given key by the provided value."""
+    if key in dictionary:
+        dictionary[key] -= value
     else:
-        dict[key] = value
-    return dict
-
+        dictionary[key] = 0 - value
+    return dictionary
 
-def subtract_or_create_key(dict, key, value):
-    if key in dict:
-        dict[key] -= value
-    else:
-        dict[key] = 0 - value
-    return dict
 
-
-def make_iaa_name_from_dict(owner):
-    return f"IAA {owner['name']}"
+def make_ma_name_from_dict(owner: Dict) -> str:
+    """Generate the name of the marget agent from the given owner."""
+    return f"MA {owner['name']}"
```

Comment:
This change affects many parts of the code across repos. This is why I needed to rename IAA strings everywhere.